jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -484,6 +486,11 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 		return String.join(prefix, suffix, delimiter, elements, size);
 	}
 	/*[ENDIF] OPENJDK_METHODHANDLES*/
+	
+	public Class<?> findBootstrapClassOrNull(String name) {
+		return VMAccess.findClassOrNull(name, ClassLoader.bootstrapClassLoader);
+	}
```

Comment:
I think this belongs in the `else` branch of the conditional block starting on line 222.

Reply:
To keep both variants working, `findBootstrapClassOrNull(ClassLoader classLoader, String name)` can be removed after current `openj9-staging` contents promoted into `openj9` branch.
Updated PR.